src/components/common/CustomCalendar.tsx

```diff
@@ -109,25 +123,33 @@ const CustomCalendar: FC<CustomCalendarProps> = (props) => {
         />
         )
         } />
+      <CustomToast
+        onHide={() => setShowFutureDateToast(false)}
+        showToast={showFutureDateToast}
+        severity="warn"
+        detail="Please select a date and time on or before the current moment."
```

Comment:
Please select a date and time from the past.

Reply:
Done